sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -4580,6 +4590,12 @@ var setParentClass = function(child, parent) {
         var fracIV = tcuInterval.applyMonotone1p(func1, iargs.a);
         var wholeIV = tcuInterval.applyMonotone1p(func2, iargs.a);
 
+        if (!iargs.a.isFinite()) {
```

Comment:
See the native code at https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#3300